lsm-perf.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env
+
+import argparse
+import time
+import plumbum
+import sys
+import statistics
+from plumbum import local, SshMachine  # TODO: requirements
+
+
+ROUNDS = 3
+REPETITIONS_PER_ROUND = 50
+WARMUP_RUNS = 5
+ON_VM_WORKLOAD_PATH = '~/lsm-perf-workload'
+QEMU_EXIT_CMD = b'\x01cq\n'  # -> ctrl+a c q
+
+
+def main(args):
+    try:
+        init_output_file(args.out)
+        for round in range(ROUNDS):
+            print('Starting round %d' % round)
+            for kernel in args.kernels:
+                results = evaluating_kernel(
+                    kernel_path=kernel.name,
+                    img_path=args.image.name,
+                    workload_path=args.workload.name,
+                    keyfile=args.key.name
+                )
+                write_results_to_file(args.out, kernel.name, round, results)
+    except KeyboardInterrupt:
+        print('\nExit prematurely')
+    finally:
+        args.out.close()
+    return 0
+
+
+def evaluating_kernel(kernel_path, img_path, workload_path, keyfile):
+    """Starts a VM with the kernel and evaluates its
+    performances on the provided workload
```

Comment:
Please format docstrings according to [PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) (sorry I didn't mention this before, I thought it was part of PEP8!)

Reply:
We should also generally have documentation for each parameter - I'd recommend doing this in Sphinx style, e.g.:

```
"""
...
:param kernel_path: Path on host to kernel image
:param workload_path: Path on host to workload executable. 
    This should be a binary that outputs a
    single number on stdout representing a benchmark score.
...
"""
```

No need to be perfect with the syntax but I've found it's useful to vaguely follow it so that if you want to set up API documentation later on in the project, it's not too much work to update all the docstrings.

Reply:
Actually PEP 257 is part of PEP8, my bad.
I'll push a commit later to fix all the documentation issues.